jit/v2/JIT_v2.0.0.yaml

```diff
@@ -893,60 +1067,446 @@ paths:
+      tags:
+        - Port Call Service
+      operationId: get-vessel-status
+      summary: Retrieves a list of **Vessel Statuses**
+      description: |
+        Retrieves a list of **Vessel Statuses**
+      responses:
+        '200':
+          description: |
+            Get the list of **Vessel Statuses** matching the specified filter
+          headers:
+            API-Version:
+              $ref: '#/components/headers/API-Version'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/VesselStatuses'
+        '500':
+          description: |
+            In case a server error occurs in implementor system a `500` (Internal Server Error) is returned
+          headers:
+            API-Version:
+              $ref: '#/components/headers/API-Version'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                internalServerErrorExample:
+                  summary: |
+                    Internal Server Error while processing Port Call Service
+                  description: |
+                    An Internal Server Error has occurred - the caller should contact {implementor-support} and provide the `providerCorrelationReference` (in the example this is `4426d965-0dd8-4005-8c63-dc68b01c4962`)
+
+                    **NB**: `errorCode` not yet standardized by DCSA. Value `7005` is just a "random example"
+                  value:
+                    httpMethod: PUT
+                    requestUri: /v2/vessel-statuses/085a3207-5e45-49cf-8e1b-f8442beaf545
+                    statusCode: 500
+                    statusCodeText: Internal Server Error
+                    statusCodeMessage: Internal Server Error occurred while processing Port Call Service
+                    providerCorrelationReference: 4426d965-0dd8-4005-8c63-dc68b01c4962
+                    errorDateTime: '2024-11-21T09:41:00Z'
+                    errors:
+                      - errorCode: 7005
+                        errorCodeText: Internal Error occurred
+                        errorCodeMessage: Internal Error occurred
+        default:
+          description: |
+            Unexpected error
+          headers:
+            API-Version:
+              $ref: '#/components/headers/API-Version'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  '/v2/port-call-services/{portCallServiceID}/decline':
+    post:
+      parameters:
+        - $ref: '#/components/parameters/portCallServiceIDPathParam'
+        - $ref: '#/components/parameters/Api-Version-Major'
+      tags:
+        - Port Call Service provider
+      operationId: decline-port-call-service
+      summary: Decline a Port Call service
+      description: |
+        Consumer declining a **Port Call Service**.
```

Comment:
Provider

Reply:
No this is correct as the decline is the opposite way (compared to cancel)